### PR TITLE
Add docstrings to filters

### PR DESCRIPTION
**Pull request by IsaacYangSLA**

### CODE REVIEW

File: nvflare/app_common/filters/exclude_vars.py [hunk 1]
```diff
@@ -23,19 +23,18 @@
 
 
 class ExcludeVars(Filter):
-    """Exclude/Remove variables from Sharable
-
-    Args:
-        exclude_vars (Union[List[str], str, None] , optional): variables/layer names to be excluded.
+    def __init__(self, exclude_vars: Union[List[str], str, None] = None):
+        """Exclude/Remove variables from Sharable
```
Comment (YuanTingHsieh) [inline nvflare/app_common/filters/exclude_vars.py:27]: typo should be Shareable?

File: nvflare/app_common/filters/svt_privacy.py [hunk 2]
```diff
@@ -22,12 +22,19 @@
...
+        lambda_rho = gamma * 2.0 / epsilon
+        threshold = tau + np.random.laplace(scale=lambda_rho)
 
-    def __init__(self, fraction=0.1, epsilon=0.1, noise_var=0.1, gamma=1e-5, tau=1e-6):
+        Args:
+            fraction (float, optional): used to determine dataset threshold. Defaults to 0.1.
+            epsilon (float, optional): Defaults to 0.1.
+            noise_var (float, optional): additive noise. Defaults to 0.1.
+            gamma ([type], optional): Defaults to 1e-5.
+            tau ([type], optional): Defaults to 1e-6.
```
Comment (YuanTingHsieh) [inline nvflare/app_common/filters/svt_privacy.py:36]: [type] should be float?

File: nvflare/app_common/filters/convert_weights.py [hunk 3]
```diff
@@ -21,14 +21,18 @@
...
     DIFF_TO_WEIGHTS = "diff_to_weights"
 
     def __init__(self, direction: str):
+        """Convert WEIGHTS to WEIGHT_DIFF or vice versa
+
+        Args:
+            direction (str): control conversion direction.  Either weights_to_diff or diff_to_weights.
+
+        Raises:
+            ValueError: [description]
```
Comment (YuanTingHsieh) [inline nvflare/app_common/filters/convert_weights.py:34]: should this [description] be changed to something else?

**Review by YuanTingHsieh — CHANGES_REQUESTED** [review]
Typos and missing types

File: nvflare/app_common/filters/exclude_vars.py (hunk 1 above)
Comment (IsaacYangSLA) [reply nvflare/app_common/filters/exclude_vars.py]: Fixed.

File: nvflare/app_common/filters/svt_privacy.py (hunk 2 above)
Comment (IsaacYangSLA) [reply nvflare/app_common/filters/svt_privacy.py]: Changed [type] to float.

File: nvflare/app_common/filters/convert_weights.py (hunk 3 above)
Comment (IsaacYangSLA) [reply nvflare/app_common/filters/convert_weights.py]: Added the description of condition when ValueError is raised.

**Pull request merged**